addons/account/models/sequence_mixin.py

```diff
@@ -159,6 +160,8 @@ def _get_last_sequence(self, relaxed=False, with_prefix=None, lock=True):
 
         :return: the string of the previous sequence or None if there wasn't any.
         """
+        if lock:
+            warnings.warn("`lock` in `_get_last_sequence` is deprecated, install `account_sequence`")
```

Comment:
```suggestion
        if lock:
            warnings.warn("`lock` in `_get_last_sequence` is deprecated, install `account_sequence`", DeprecationWarning, stacklevel=2)
```
Also seems rather inconvenient that the default behaviour is deprecated.

Reply:
It is not the default behavior anymore